vgscheckout/src/androidTest/java/com/verygoodsecurity/vgscheckout/util/AccessTokenHelper.kt

```diff
@@ -0,0 +1,45 @@
+package com.verygoodsecurity.vgscheckout.util
+
+import com.google.gson.JsonParser
+import com.verygoodsecurity.vgscheckout.BuildConfig
+import okhttp3.*
+import okhttp3.RequestBody.Companion.toRequestBody
+
+class AccessTokenHelper {
```

Comment:
@Endorf Can we make the `AccessTokenHelper` an object? So we will be able to remove the companion.